t/notes.t

```diff
@@ -0,0 +1,27 @@
+use strict;
+use warnings;
+use utf8;
+
+use lib 't/lib';
+
+use Apache::Test qw(-withtestmore);
+use Apache::TestRequest;
+use JSON::XS qw( decode_json );
+use Test::ModMaxMindDB;
+
+my $url = '/cgi-bin/valid-db/json-env';
```

Comment:
Should this be hitting /notes? I don't see us actually hitting the new path, nor checking what the test package is doing.

Reply:
That explains why it surprisingly passed right away. :rofl: